src/components/Profile/EditProfileForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component, Fragment } from 'react';
+import { Form, Button, Card, Header } from 'semantic-ui-react';
+import { getProfile } from '../../redux/actioncreators/userProfile';
+import { editProfile } from '../../redux/actioncreators/editProfile';
+import { connect } from 'react-redux';
+import './profile.scss';
+
+export class EditProfileForm extends Component {
+    state = {
+        name: '',
+        bio: '',
+        image: ''
+    }
+    componentDidMount() {
+        const { getProfile, profile } = this.props;
+        getProfile()
+        this.setState({
+            username: profile.username,
+            bio: profile.bio,
+            image: profile.image,
+        });
+    }
+    
+    componentWillMount() {
```

Comment:
I do not think that we need componentWillMount(). When fetching data, it is preferred to use componentDidMount(). 
This method will fetch data even before the first render.

Reply:
Noted working to change it right now